rest-api-spec/src/main/resources/rest-api-spec/api/delete_scroll.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
no you are right. it is a leftover, due to the fact that I thought POST should support the query_string parameter at all, now that it does, I can get right of this new api.